src/schema/v2/partner.ts

```diff
@@ -98,6 +100,7 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
       },
       artworksConnection: {
         description: "A connection of artworks from a Partner.",
+        deprecationReason: "Use `filterArtworksConnection`",
```

Comment:
@dzucconi question: 

`artworksConnection` hits `partner/{id}/artworks`, while `filterArtworksConnection` hits `filter/artworks`. PX is in the process of implementing a new version of the "todo" checklist on CMS home, and we're planning to use `partner { artworksConnection {...} }` and adding some specific filtering to [that endpoint in Gravity](https://github.com/artsy/gravity/pull/13790).

Any objections to un-deprecating this?

Reply:
The filterArtworksConnection on partner passes `partner_id` to scope. Feel free to un-deprecate but I guess the filter situation is much worse than I previously feared.

Realistically, I'd like us to move to a single, top level filter field. Supporting these nested fields makes our types and components deeply complicated for no real benefit.

Reply:
I'm not familiar with the aforementioned filter situation - do you mean that there are many different ways we're currently able to query for artworks? Asking for my own edification.

Regarding a single, top-level filter field, you mean like a unified field so that any time we're querying for artworks from anywhere we're always using the same field (and it would take a wide variety of params to support our many use-cases)?

Reply:
Yes; like this was/is a problem because you can't use any of the filter UI with the previous nested filter endpoint because it doesn't support any of the same arguments.

A single top-level field with a wide array of arguments modeled as actual graphql input objects would allow our UI to be truly re-suable instead of the situation currently which is unmaintainable: constant copy pasting of verbose typed arguments, some nested fields not supporting arguments, etc.

Reply:
I had to expose a proper filter connection here so as to support filters on partner screens on iOS. Similarly; once we re-build the partner pages in Force we'll have to use that as well.

Reply:
I think there's a valid use case to have both of these currently (although the naming is confusing). `artworksConnection` is the Mongo-backed one and `filterArtworksConnection` is the ES-backed one. There is some overlap, but each of them offers things the other does not.

In an ideal world having a simpler schema (maybe a root `artworksConnection` that can accept all combinations, etc.) would be nice, but for now both should exist.

Reply:
Ah, I definitely didn't even know we had a mongo backed filter connection

Reply:
Thanks for clarifying why we have both. Confirming that we'll undeprecate `artworksConnection` and keep in mind that our goal end state is to have one centralized field.